### PR TITLE
github(summary): update learn more link

### PR DESCRIPTION
**Pull request by crazy-max**
updates learn more link to the docker blog post https://www.docker.com/blog/new-beta-feature-deep-dive-into-github-actions-docker-builds-with-docker-desktop/

### CODE REVIEW

File: src/github.ts [hunk 1]
```diff
@@ -240,7 +240,7 @@ export class GitHub {
         .addRaw(`For a detailed look at the build, download the following build record archive and import it into Docker Desktop's Builds view. `)
         .addBreak()
         .addRaw(`Build records include details such as timing, dependencies, results, logs, traces, and other information about a build. `)
-        .addRaw(addLink('Learn more', 'https://docs.docker.com/go/build-summary/'))
+        .addRaw(addLink('Learn more', 'https://www.docker.com/blog/new-beta-feature-deep-dive-into-github-actions-docker-builds-with-docker-desktop/?utm_source=github&utm_medium=actions'))
```
Comment (crazy-max) [inline src/github.ts:243]: also sets `utm_source` and `utm_medium`

Comment (colinhemmings) [reply src/github.ts:243]: Go idea to add the UTM components. Do you think we should still use a go path and handle the redirect?
@dvdksn, maybe we should distill this post down into some "Introductory Guide" for users not familiar with Builds View?

Comment (dvdksn) [reply src/github.ts:243]: Hmm yeah, why not? We could have that intro page in the manual section for builds, and link off to the full description in the Docker Desktop section.

Comment (crazy-max) [reply src/github.ts:243]: @colinhemmings 

> Do you think we should still use a go path and handle the redirect?

I don't think `/go` fits if we are using UTM parameters

Comment (dvdksn) [reply src/github.ts:243]: We have a few `go`-links with utm params, for example:
https://github.com/docker/docs/blob/main/data/redirects.yml#L694-L700

Comment (crazy-max) [reply src/github.ts:243]: > We have a few `go`-links with utm params, for example: https://github.com/docker/docs/blob/main/data/redirects.yml#L694-L700

How do we make sure the source/medium is right? For example if I open within my browser `https://docs.docker.com/go/build-eap/` these are wrong are this does not come from Docker Desktop but directly in my browser. That's my concern.

Comment (dvdksn) [reply src/github.ts:243]: yeah, there is no guarantee that the link will be accessed only from the correct source, providing the correct utm.

**Pull request merged**